python/ray/serve/tests/unit/test_asyncio.py

```diff
@@ -0,0 +1,76 @@
+import sys
```

Comment:
nit: I'd name the test something more specific like `test_run_coroutine_threadsafe`

hopefully we don't end up in a habit of vendoring more asyncio code 😅